client/signup/navigation-link/style.scss

```diff
@@ -4,3 +4,7 @@
 	margin: 24px auto;
```

Comment:
I think that I would just make this `margin: 24px;` and not add the new rule. Then we won't need to account for a change in RTL, any possibility that they aren't exactly centered, or if there were ever more than 2 buttons.